repo/packages/M/mongodb-admin/0/resource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "images": {
+    "icon-small": "http://www.file-extensions.org/imgs/app-icon/48/8854/mongodb-icon.png",
+    "icon-medium": "http://www.sudops.com/wp-content/uploads/2014/02/mongodb-96x96.png",
+    "icon-large": "http://1.bp.blogspot.com/-FagQLEI_Zhc/VnAzE9L8e3I/AAAAAAAAAFk/DWPqvL0ZXo8/s320/mongodb.jpg",
+    "screenshots": [
+      "https://raw.githubusercontent.com/mrvautin/mrvautin.github.io/master/images/adminMongo/adminMongo_collectionview.png",
+      "https://raw.githubusercontent.com/mrvautin/mrvautin.github.io/master/images/adminMongo/adminMongo_dbview.png",
+      "https://raw.githubusercontent.com/mrvautin/mrvautin.github.io/master/images/adminMongo/adminMongo_connections.png"
+    ]
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "mongodb-admin-docker": "mrvautin/adminmongo:latest"
```

Comment:
Understood, I try to avoid it but this image doesn't offer other tags: https://hub.docker.com/r/mrvautin/adminmongo/tags/

we could fork it and tag it if that's something we want to do. Again, I guess that should be in some sort of mesosphere account?